cfg/converter/games/de2/version_hashes.toml

```diff
@@ -1,16 +1,16 @@
 # version hashes of Age of Empires 2: Definitive Edition
 
-file_version = "1.0"
+file_version = "2.0"
 hash_algo = "SHA3-256"
 
 [AoE2DE_s]
-path = "AoE2DE_s.exe"
+paths = ["AoE2DE_s.exe", "AoE2DE.exe"]
```

Comment:
Thank you! <3

Reply:
@duanqn I totally forgot to ask for the hashes of your `AoE2DE.exe` file :D Shouldn't be a problem though because the hash checks are not mandatory.